SU2_CFD/src/integration/CIntegration.cpp

```diff
@@ -331,12 +331,12 @@ void CIntegration::SetStructural_Solver(CGeometry *geometry, CSolver **solver_co
     case (CD_EXPLICIT):
       break;
     case (NEWMARK_IMPLICIT):
-      if (fsi) solver_container[FEA_SOL]->ImplicitNewmark_Relaxation(geometry, solver_container, config);
+      if (fsi) solver_container[FEA_SOL]->ImplicitNewmark_Relaxation(geometry, config);
       break;
     case (GENERALIZED_ALPHA):
-      //if (fsi)  solver_container[FEA_SOL]->Update_StructSolution(geometry, solver_container, config);
-      solver_container[FEA_SOL]->GeneralizedAlpha_UpdateSolution(geometry, solver_container, config);
-      solver_container[FEA_SOL]->GeneralizedAlpha_UpdateLoads(geometry, solver_container, config);
+      //if (fsi)  solver_container[FEA_SOL]->Update_StructSolution(geometry, config);
```

Comment:
delete comment if not needed

Reply:
I'm not the one who commented, so I don't really know if it is needed. The generalized alpha method is not fully implemented, at least in some areas it prints "NOT IMPLEMENTED" messages, I would rather leave it like this until the implementation is finished.

Reply:
Got it, sounds good. Can it just be uncommented then if it throws a proper error? Would be less confusing for the next person who comes across it

Reply:
I ended up removing the method entirely, if that operation is needed it can be moved to `GeneralizedAlpha_UpdateSolution`